.github/workflows/docker-scan.yml

```diff
@@ -0,0 +1,32 @@
+name: "docker-scan"
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 1'
+  watch:
+    types: [started]
+
+jobs:
+  Trivy-Scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t docker.pkg.github.com/ossf/package-feeds:${{ github.sha }} .
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```

Comment:
Can we pin this to a version or commit? It's my understanding that using actions from master isn't really recommended, if possible (even though this particular one looks lower risk).